requirements/requirements-dev.example.txt

```diff
@@ -0,0 +1 @@
+# Additional dev-only requirements should be here (e.g. debuggers, ipython...)
```

Comment:
Based on [pip-tools documentation](https://github.com/jazzband/pip-tools#workflow-for-layered-requirements), I think we can use the below lines to automatically include `requirements-base.txt` (production packages list) in `requirements-dev.txt`.
By this change, we can only install `requirements-dev.txt` in the development environment.
```
-c requirements.txt
... development packages
```
By using this feature of pip-tools we can have layer based requirements files:
```
# requirements.txt
... production packages
```
```
# requirements-testing.txt
... production packages
... testing packages
```
```
# requirements-dev.txt
... production packages
... testing packages
... development packages
```
And, in the end, I think we can put `pre-commit` in `requirements-dev.txt` bcause we use it only in development.